slices/libmagic-mgc.yaml

```diff
@@ -0,0 +1,13 @@
+package: libmagic-mgc
+
+essential:
+  - libmagic-mgc_copyright
+
+slices:
+  libs:
```

Comment:
I think we should name this slice `data` instead of `libs` since it's not actually a shared library loaded by the system's dynamic linker.